bin/install-if-deps-outdated.js

```diff
@@ -24,7 +24,7 @@ const needsInstall = () => {
 };
 
 if ( needsInstall() ) {
-	const installResult = spawnSync( 'npm', [ 'ci' ], {
+	const installResult = spawnSync( 'npx', [ 'lerna', 'bootstrap' ], {
```

Comment:
Before, we expected root shrinkwrap to exist, it seems that now we should expect package locks to exist and we could continue to use `CI`.
```suggestion
	const installResult = spawnSync( 'npx', [ 'lerna', 'bootstrap', '--ci' ], {
```